w3af/core/controllers/dependency_check/platforms/ubuntu1410.py

```diff
@@ -0,0 +1,72 @@
+"""
+ubuntu1410.py
+
+Copyright 2013 Andres Riancho
+
+This file is part of w3af, http://w3af.org/ .
+
+w3af is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation version 2 of the License.
+
+w3af is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with w3af; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+"""
+import subprocess
+import platform
+
+from .base_platform import Platform
+from ..requirements import CORE, GUI
+
+
+class Ubuntu1410(Platform):
```

Comment:
Why don't you inherit from `Ubuntu1404`? That would make the `Ubuntu1410` class really small, avoid code duplication at `os_package_is_installed`, etc